README.md

```diff
@@ -1 +1,53 @@
-# crypto-hub
\ No newline at end of file
+# REACT CAPSTONE
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Redux
+- React
+## Sample images
+![screenshot](./src/images/Screenshot1.png)
+![screenshot](./src/images/Screenshot2.png)
+
+
+
+## Live previewhttps://deploy-preview-6--sympho
+[preview](https://crypto-hub1111.herokuapp.com/)
```

Comment:
- Well done to add the presentation video as the other [code reviewer said](https://github.com/Richez003/crypto-hub/pull/1#:~:text=There%20should%20be,proud%20of%20making.).